### PR TITLE
[Doc]--remove pulsarctl from pulsar docs

### LINKED ISSUE #8400: [doc] The command bin/pulsarctl cannot be used in the offloader document
**Describe the bug**
The command bin/pulsarctl cannot be used in the offloader document.

**To Reproduce**
1. Go to 'https://pulsar.apache.org/docs/en/next/tiered-storage-filesystem/#example-2'
2. The command `bin/pulsarctl`  cannot be found in the PULSAR_HOME/bin

### PR DESCRIPTION
**Pull request by Huanli-Meng**
Fixes #8400 

### Motivation

pulsarctl is not supported for pulsar docs. Therefore, remove pulsarctl commands and code examples

### Modifications

use pulsar-admin commands and code examples to replace pulsarctl commands and code examples.

### CODE REVIEW

**Review by sijie — APPROVED** [review]

File: site2/docs/tiered-storage-filesystem.md [hunk 1]
```diff
@@ -218,12 +218,12 @@ To trigger via CLI tools, you need to specify the maximum amount of data (thresh
 
     > #### Tip
     >
-    > For more information about the `pulsarctl topic offload options` command, including flags, descriptions, default values, and shorthands, see [here](https://streamnative.io/docs/pulsarctl/v0.4.0/#-em-offload-em-). 
+    > For more information about the `pulsar-admin topics offload options` command, including flags, descriptions, default values, and shorthands, see [here](https://pulsar.apache.org/docs/en/next/pulsar-admin/#offload).
```
Comment (Jennifer88huang) [inline site2/docs/tiered-storage-filesystem.md:221]: use relative path
use "(reference-pulsar-admin.md)" in the link

File: site2/docs/tiered-storage-filesystem.md [hunk 2]
```diff
@@ -261,7 +261,7 @@ To trigger via CLI tools, you need to specify the maximum amount of data (thresh
 
     > #### Tip
     >
-    > For more information about the `pulsarctl topic offload-status options` command, including flags, descriptions, default values, and shorthands, see [here](https://streamnative.io/docs/pulsarctl/v0.4.0/#-em-offload-status-em-). 
+    > For more information about the `pulsar-admin topics offload-status options` command, including flags, descriptions, default values, and shorthands, see [here](https://pulsar.apache.org/docs/en/next/pulsar-admin/#offload-status).
```
Comment (Jennifer88huang) [inline site2/docs/tiered-storage-filesystem.md:264]: use relative path.

**Comment by Huanli-Meng** [comment]
@Jennifer88huang , Thanks a lot for your comments. Docs have been updated based on your comments. Please help double check them. Thanks.

**Review by Jennifer88huang — APPROVED** [review]
well done

**Pull request merged**